diff --git a/node_modules/broccoli-babel-transpiler/index.js b/node_modules/broccoli-babel-transpiler/index.js
index f27d48e..4087897 100644
--- a/node_modules/broccoli-babel-transpiler/index.js
+++ b/node_modules/broccoli-babel-transpiler/index.js
@@ -55,7 +55,7 @@ function Babel(inputTree, _options) {
   delete options.browserPolyfill;
   delete options.browserPolyfillPath;
 
-  Filter.call(this, inputTree, options);
+  new Filter(inputTree, options);
 
   delete options.persist;
   delete options.async;
