diff --git a/.npmignore b/.npmignore
deleted file mode 100644
index 135c7b1f27a45d59ccd18048afaa5b4b06f38e0a..0000000000000000000000000000000000000000
diff --git a/index.js b/index.js
index 6000638f248f782ec333ae40ee2a015bd676fe39..2850acf737d89a5f87f2a25e60e83efb817cc4f3 100644
--- a/index.js
+++ b/index.js
@@ -24,7 +24,7 @@ module.exports = {
     app.import("vendor/ember-cli-jstree/style.css");
     app.import("vendor/ember-cli-jstree/register-version.js");
   },
-
+/*
   treeForVendor(tree) {
     let registerVersionTree = writeFile(
       "ember-cli-jstree/register-version.js",
@@ -47,7 +47,7 @@ module.exports = {
 
     return tree ? mergeTrees([tree, registerVersionTree, jsTree, stylesTree], {overwrite: true}) : mergeTrees([registerVersionTree, jsTree, stylesTree], {overwrite: true});
   },
-
+*/
   treeForPublic() {
     return new Funnel(path.join(this._jstreePath(), "dist/themes/default"), {
       include: ["**/*.png", "**/*.gif"],
diff --git a/yarn.lock b/yarn.lock
deleted file mode 100644
index 0d139ab61d6a8345f7471529065d646cf3c77b06..0000000000000000000000000000000000000000