diff --git a/node_modules/ember-cli-shims/index.js b/node_modules/ember-cli-shims/index.js
index 77a3496..bdc5b1f 100644
--- a/node_modules/ember-cli-shims/index.js
+++ b/node_modules/ember-cli-shims/index.js
@@ -27,10 +27,10 @@ module.exports = {
       throw new SilentError('To consume ember-cli-shims from npm you must be using ember-cli@2.11.0-beta.1 or greater.  Please update ember-cli to a newer version or remove ember-cli-shims from `package.json`.');
     }
 
-    var projectBowerDeps = this.project.bowerDependencies();
+    /* var projectBowerDeps = this.project.bowerDependencies();
     if (projectBowerDeps['ember-cli-shims']) {
       throw new SilentError('Using ember-cli-shims as both a bower dependency and an npm dependency is not supported. Please remove `ember-cli-shims` from `bower.json`.');
-    }
+    } */
 
     // ember-source@2.11.0-alpha and 2.11.0-beta series releases included
     // their own legacy shims system, so this import is not needed with
